dbms/src/Storages/KVStore/FFI/ProxyFFI.h

```diff
@@ -234,7 +236,7 @@ inline EngineStoreServerHelper GetEngineStoreServerHelper(EngineStoreServerWrap
         .fn_set_store = SetStore,
         .fn_set_pb_msg_by_bytes = SetPBMsByBytes,
         .fn_handle_safe_ts_update = HandleSafeTSUpdate,
-        .fn_fast_add_peer = FastAddPeer,
+        .fn_fast_add_peer = FastAddPeer
```

Comment:
nit: better keep the trailing `,` at the last struct member, or sometime clang-format will merge the line 240 `};` into line 239